packages/admin/dashboard/src/i18n/translations/en.json

```
@@ -912,6 +912,7 @@
     "exchange": "Exchange",
     "return": "Return",
     "cancelWarning": "You are about to cancel the order {{id}}. This action cannot be undone.",
+    "orderCancelled": "Order cancelled successfully",
```

todo: should use the american version of cancel
```suggestion
    "orderCanceled": "Order canceled successfully",
```